source/common/http/date_provider_impl.h

```diff
@@ -35,6 +35,7 @@ class TlsCachingDateProviderImpl : public DateProviderImplBase, public Singleton
   TlsCachingDateProviderImpl(Event::Dispatcher& dispatcher, ThreadLocal::SlotAllocator& tls);
 
   // Http::DateProvider
+  void setDateHeader(RequestHeaderMap& headers) override;
```

Comment:
Looks like these (and below) are the same implementation. Why not use `RequestOrResponseHeaderMap` as a param?

Reply:
Oh nice, didn't realize that was a thing. Updated!